README.md

```diff
@@ -63,3 +63,58 @@ Google Sign-In also supports iOS apps that are built for macOS via
 [Mac Catalyst](https://developer.apple.com/mac-catalyst/).  In order for your Mac Catalyst app
 to store credientials via the Keychain on macOS, you will need to
 [sign your app](https://developer.apple.com/support/code-signing/).
+
+## Using the Google Sign-In Button
+
+Depending upon your platform, GoogleSignIn-iOS provides a few options to use
```

Comment:
"There are several paths to adding a 'Sign in with Google' button to your app, which path you choose will depend on your UI framework and target platform."